Src/CSharpier/SyntaxPrinter/SyntaxNodePrinters/Parameter.cs

```diff
@@ -10,24 +10,37 @@ public static class Parameter
     {
         public static Doc Print(ParameterSyntax node)
         {
+            var hasAttribute = node.AttributeLists.Any();
+            var groupId = hasAttribute ? Guid.NewGuid().ToString() : string.Empty;
             var docs = new List<Doc>
             {
                 AttributeLists.Print(node, node.AttributeLists),
-                node.AttributeLists.Any() ? Doc.Line : Doc.Null,
+                hasAttribute ? Doc.IndentIfBreak(Doc.Line, groupId) : Doc.Null,
                 Modifiers.Print(node.Modifiers)
             };
+
+            var paramDocs = new List<Doc>();
             if (node.Type != null)
             {
-                docs.Add(Node.Print(node.Type), " ");
+                paramDocs.Add(Node.Print(node.Type), " ");
             }
 
-            docs.Add(Token.Print(node.Identifier));
+            paramDocs.Add(Token.Print(node.Identifier));
             if (node.Default != null)
             {
-                docs.Add(EqualsValueClause.Print(node.Default));
+                paramDocs.Add(EqualsValueClause.Print(node.Default));
             }
 
-            return Doc.Concat(docs);
+            if (hasAttribute)
+            {
+                docs.Add(Doc.Concat(paramDocs));
+                return Doc.GroupWithId(groupId, docs.ToArray());
```

Comment:
Just realized @shocklateboy92 had a similar idea and created an issue for it https://github.com/belav/csharpier/issues/334
So this can just get updated when someone picks up that issue.

Reply:
I could solve that issue as part of this, not a big issue to do so.